src/content/peerconnection/audio/js/main.js

```diff
@@ -359,3 +367,33 @@ window.setInterval(() => {
     lastResult = res;
   });
 }, 1000);
+
+if (window.RTCRtpReceiver && ('getSynchronizationSources' in window.RTCRtpReceiver.prototype)) {
```

Comment:
I believe getSynchronizationSources has been available on all major browsers for several years, do we really need to guard it at this point?

Reply:
you should principally guard in real-world applications.